org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/lsp/JavaProtocolExtensions.java

```diff
@@ -66,6 +67,13 @@ public interface JavaProtocolExtensions {
 	@JsonNotification
 	void projectConfigurationUpdate(TextDocumentIdentifier documentUri);
```

Comment:
Just wanted to confirm that the reason to duplicate this is mainly to avoid a potentially breaking API change for clients. My first though would have been to merge these and respresent the case of 1 document as simply a 1-element array being sent over.

If that's the case, we can leave this as it is.

Reply:
Yes, it's to avoid breaking other clients. Maybe I should add deprecated comments to the original one and encourage other clients migrate to the new one?